### ISSUE TITLE
DeepMI: A Mutual Information Based Framework For Unsupervised Deep Learning of Tasks. (arXiv:2101.06411v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#2902]
DeepMI: A Mutual Information Based Framework For Unsupervised Deep Learning of Tasks. (arXiv:2101.06411v1 [cs.CV])<br>
https://ift.tt/35VZZ9g<br>
<p>In this work, we propose an information theory based framework DeepMI to train deep neural networks (DNN) using Mutual Information (MI). The DeepMI framework is especially targeted but not limited to the learning of real world tasks in an unsupervised manner. The primary motivation behind this work is the insufficiency of traditional loss functions for unsupervised task learning. Moreover, directly using MI for the training purpose is quite challenging to deal because of its unbounded above nature. Hence, we develop an alternative linearized representation of MI as a part of the framework. Contributions of this paper are three fold: i) investigation of MI to train deep neural networks, ii) novel loss function LLMI, and iii) a fuzzy logic based end-to-end differentiable pipeline to integrate DeepMI into deep learning framework. We choose a few unsupervised learning tasks for our experimental study. We demonstrate that L LM I alone provides better gradients to achieve a neural network better performance over the cases when multiple loss functions are used for a given task.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
